module2/src/com/epam/arraysofarrays/MatrixSorter.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.arraysofarrays;
+
+public class MatrixSorter {
+    //task 12
+    public static int[][] sortRowsAscendingOrDescending(int[][] matrix, boolean ascending) {
+        if (ascending) {
+            for (int k = 0; k < matrix.length; k++) {
+                for (int i = 0; i < (matrix.length - 1); i++) {
+                    for (int j = 0; j < matrix.length - i - 1; j++) {
+                        if (matrix[k][j] > matrix[k][j + 1]) {
+                            int swapped = matrix[k][j];
+                            matrix[k][j] = matrix[k][j + 1];
+                            matrix[k][j + 1] = swapped;
+                        }
+                    }
+                }
+            }
+            return matrix;
+        } else {
+            for (int k = 0; k < matrix.length; k++) {
+                for (int i = 0; i < (matrix.length - 1); i++) {
+                    for (int j = 0; j < matrix.length - i - 1; j++) {
+                        if (matrix[k][j] < matrix[k][j + 1]) {
+                            int swapped = matrix[k][j];
+                            matrix[k][j] = matrix[k][j + 1];
+                            matrix[k][j + 1] = swapped;
+                        }
+                    }
+                }
+            }
+            return matrix;
+        }
+    }
```

Comment:
You asked me about the best solution, check this:
```suggestion
    // task_12
    public static void sortRows(int[][] matrix, boolean descendingOrder) {
        for (int[] row : matrix) {
            Arrays.sort(row);
        }
        if (descendingOrder) {
            for (int i = 0; i < matrix.length; i++) {
                int lastItemIndex = matrix[i].length - 1;
                int middleOfRow = matrix[i].length / 2;
                int tempItem;
                for (int j = 0; j < middleOfRow; j++) {
                    tempItem = matrix[i][j];
                    matrix[i][j] = matrix[i][lastItemIndex - j];
                    matrix[i][lastItemIndex - j] = tempItem;
                }
            }
        }
    }
```